tests/Doctrine/Tests/ODM/PHPCR/Functional/Translation/AttributeTranslationStrategyTest.php

```diff
@@ -167,13 +171,20 @@ public function testTranslationNullProperties()
         $this->assertEquals('Some interesting subject', $doc->topic);
         $this->assertEquals('Lorem ipsum...', $doc->getText());
         $this->assertEquals('not null', $doc->nullable);
+        $this->assertEquals('Custom property name', $doc->customPropertyName);
         $this->assertEquals(array('key' => 'value'), $doc->getSettings());
 
         $strategy->loadTranslation($doc, $node, $this->metadata, 'fr');
         $this->assertEquals('Un sujet intéressant', $doc->topic);
         $this->assertEquals('Lorem français', $doc->getText());
         $this->assertNull($doc->nullable);
+        $this->assertNull($doc->customPropertyName);
         $this->assertEquals(array(), $doc->getSettings());
+
+        $nullFields = $node->getProperty('phpcr_locale:fr' . AttributeTranslationStrategy::NULLFIELDS)->getValue();
+        $this->assertEquals(array(
+            'custom-property-name',
```

Comment:
would it not be sufficient to have an extra single multivalue field which lists the active translations? I guess its done this way beacause of merging ? /cc @uwej711